Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game-page/play-game.component.html

```diff
@@ -1,5 +1,5 @@
-<label id="balance">Your Balance: {{player.balance}}</label>
-<div id="wagerOptions">
+<label id="balance">Your Balance: {{user.balance}}</label>
+<div id="wagerOptions" [class.hidden]="!isHidden">
```

Comment:
I did it like this for the wager div and play button while the playarea div is the same as isHidden. This let me use a single "toggle" for the three. If its better for me to instead differentiate between the two like I did with game and play again buttons I can do that. Something like "isSetupHidden" for the wager div and play button, then change the play div to "isPlayAreaHidden". I suppose I did it that way out of curiosity so I can easily adjust

Reply:
Yeah, I think in this case separation of responsibility is a good thing.

The reason being that if you centralize it all under one variable like it is right now it causes some things to read in counterintuitive ways. 

If centralizing everything under one variable lead to straightforward code I'd say go for it. 

But because centralizing has lead to the code that it has I would personally advocate for separation of responsibility and two variables